projects/storefrontstyles/scss/components/layout/_skip-link.scss

```diff
@@ -0,0 +1,26 @@
+%cx-skip-link {
+  button {
+    opacity: 0;
+    position: fixed;
+    top: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    background-color: var(--cx-color-light);
+    color: var(--cx-color-dark);
+    font-weight: bold;
+    padding: 16px 32px 16px 32px;
+    border-radius: 0 0 5px 5px;
+    border: 2px solid var(--cx-color-dark);
```

Comment:
should we add `border-top-width:0;`, or is the border on top of the skiplinks on purpose?